src/scenes/home/landing/hero/hero.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import LinkButton from 'shared/components/linkButton/linkButton';
+import commonUrl from 'shared/constants/commonLinks';
+import styles from './hero.css';
+
+class Hero extends Component {
+  state = {
+    donateClass: null
+  };
+  componentDidMount() {
+    if (window.innerWidth < 780) {
+      this.setState({
+        donateClass: 'bordered-white-small'
+      });
+    } else {
+      this.setState({
+        donateClass: 'bordered-dark-blue-small'
+      });
+    }
+  }
+
+  render() {
+    window.onresize = () => {
+      if (window.innerWidth < 780) {
+        this.setState({
+          donateClass: 'bordered-white-small'
+        });
+      } else {
+        this.setState({
+          donateClass: 'bordered-dark-blue-small'
+        });
+      }
+    };
+    return (
+      <div className={styles.heroContainer}>
+        <div className={styles.pageHeading}>
+          <h1>Deploy the future!</h1>
```

Comment:
Please remove the `!` from the end of the `h1`.

Reply:
The 'Deploy the future' text would look nice as a non-bolded font-weight (to match the font-style in the design).